turbo/mixins.py

```diff
@@ -56,13 +32,42 @@ def broadcast(self, model_action):
             streams_action = REMOVE
         else:  # TODO: What should the default be?
             streams_action = REPLACE
+        return streams_action
+
+    def broadcast(self, model_action):
+        streams_action = self.get_action(model_action)
 
         if self.broadcast_self:
-            self.broadcast_to_instance(self, streams_action, to_list=False)
+            self.send_broadcast(self, streams_action)
 
         for field_name in self.broadcasts_to:
             if hasattr(self, field_name):
-                self.broadcast_to_instance(getattr(self, field_name), streams_action, to_list=True)
+                self.send_broadcast(getattr(self, field_name), streams_action)
+            else:
+                self.send_broadcast(field_name, streams_action)
+
+    def _get_context(self):
+        app_name, model_name = self._meta.label.lower().split(".")
+        return {
+            "model_template": self.get_turbo_streams_template(),
+            "object": self,
```

Comment:
@davish regarding this line... should we simply do all the template rendering in the `send_broadcast` method? Then we get rid of de-/serialization. WDYT?

Reply:
I know that the rails integration purposely does the rendering on an asynchronous thread so that any latency from rendering the template doesn't affect the save action on the model. Up until now, I was sticking with that, but I also don't have too much experience with big django templates to know how slow/fast rendering can be.